Tryouts/Prototypes/Shell/MainWebWindowOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shell
+{
+    public sealed class MainWebWindowOptions
+    {
+        public string Title { get; set; }
```

Comment:
If any of these properties are optional, they should have nullable type like `string?` or `int?`

Reply:
since Window.Width and Height are double (fixed the wrong type now) instead of double? I can't set it double? here because I'd need to cast where I assign it.  These are never null because the default value is 0.0. 
And if the user doesn't specify the value from the JavaScript API the window is set to 450x800 now.